src/wfs/src/main/java/org/geoserver/wfs/WFSInfoImpl.java

```diff
@@ -21,6 +21,7 @@
     protected boolean encodeFeatureMember = false;
     protected boolean hitsIgnoreMaxFeatures = false;
     protected List<String> srs = new ArrayList<String>();
+    protected boolean allowGlobalQueries = true;
```

Comment:
Same observation made in the global setting apply here.

Reply:
Applied.

Reply:
Thanks for the fix. Sorry, noticed just now that the hashcode and equals implementations were not updated to include the new field, here too.